spring-pulsar/src/main/java/org/springframework/pulsar/core/PulsarAdministration.java

```diff
@@ -89,17 +88,20 @@ public void setApplicationContext(ApplicationContext applicationContext) throws
 
 	private void loadConf(PulsarAdminBuilder builder, Map<String, Object> adminConfig) {
 		Map<String, Object> conf = new HashMap<>(adminConfig);
-		if (conf.remove("connectTimeout") instanceof Duration connectTimeout) {
-			builder.connectionTimeout((int) connectTimeout.toMillis(), TimeUnit.MILLISECONDS);
+
+		// Workaround the fact that the PulsarAdminImpl does not attempt to construct the
```

Comment:
I would like to move this to PulsarAdminBuilderImpl.loadConf like we did for the auth stuff below.

Reply:
Added https://github.com/spring-projects-experimental/spring-pulsar/issues/245 to track replacement once fixed in Pulsar 2.12.